notebooks-auth/test_cowbird_jupyter.ipynb

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        !jhubctl run --hub "https://{PAVICS_HOST}/jupyter/" --auth-type token -u {TEST_USER} -n resources/user_test_cowbird_jupyter.ipynb \
</pre><p>Use a <code>{JUPYTER_URL}</code> and define <code>JUPYTERHUB_URL=f"{JUPYTER_URL}/hub/"</code> ?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>